src/bokeh/core/property/color.py

```diff
@@ -131,7 +132,7 @@ def __init__(self, default: Init[str | tuple[int, int, int] | tuple[int, int, in
                  Tuple(Byte, Byte, Byte),
                  Tuple(Byte, Byte, Byte, Percent),
                  RGB)
-        help = f"{help or ''}\n{self._default_help}"
+        help = f"{help or ''}\n{dedent(self._default_help)}"
```

Comment:
@mosc9575 I don't think we do this in other properties, what is the motivation here? Will this make things consistent or inconsistent with other properties help display? If the former, is there another fix that would work without adding explicit `dedents`? If the latter, we probably should hold off for a general fix if something needs fixing generally

Reply:
The motivation is again a not well indented code block. See the [property_mixins](https://docs.bokeh.org/en/latest/docs/reference/core/property_mixins.html#bokeh.core.property_mixins.FillProps.fill_color) page. The `default_help` is added including the whitespaces. It is possible to write the multiline string like 

```
    _default_help = """\
Acceptable values are:

- any of the |named CSS colors|, e.g ``'green'``, ``'indigo'``
- RGB(A) hex strings, e.g., ``'#FF0000'``, ``'#44444444'``
- CSS4 color strings, e.g., ``'rgba(255, 0, 127, 0.6)'``,
...
```

but I didn't like this. I don't have other ideas.